resources/glossary/machine-learning.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Machine Learning
+description: general introduction to how Machine Learning could be applied in the context of multi-agent simulations
+slug: machine-learning
+---
+
+## WHAT IS MACHINE LEARNING
+
+**Machine Learning** is a sub-field of Artificial Intelligence where algorithms learn from interacting with an **environment** or from a **training dataset**. Machine Learning is primarily used to make **predictions** using statistics, applying those statitical methods to business problems is the realm of **predictive analytics**. Most recent advances in Machine Learning have been driven by research in Deep Learning, which is the combination of Machine Learning and multi-layer [artificial neural networks](https://hash.ai/wiki/neural-nets).
+
+<p align="center">
+![Artificial Intelligence Machine Learning Deep Learning](https://i.imgur.com/xatMpJa.png)
```

Comment:
1. We should move images into the GitHub repo for this PR to merge.
2. I've captured a followup card to redesign this illustration in HASH house style, but this second bullet shouldn't block merge.

Reply:
Makes sense. I haven't found any place to insert images for now. What do you mean by HASH house style? Using the same palette as hash.ai's logo / landing (blue/pink)?

Reply:
Re: best place to upload resource-site imagery, I've opened an internal thread under #simulations w/ Ciaran.

By style I mean in keeping with https://cdn-us1.hash.ai/site/hash-des-flow.svg (from https://hash.ai/glossary/discrete-event-modeling). There are some other figures within the glossary that also need redoing (e.g. https://hash.ai/glossary/system-dynamics) so we shouldn't consider the mismatch a blocker. I've captured the followup, so no action required on your part here.